app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java

```diff
@@ -461,8 +460,13 @@ public void handleResult(@NonNull final ChannelInfo result) {
         if (menuRssButton != null) {
             menuRssButton.setVisible(!TextUtils.isEmpty(result.getFeedUrl()));
         }
-
-        playlistControlBinding.getRoot().setVisibility(View.VISIBLE);
+        //PlalistControls should be visible only if there is some item in
```

Comment:
```suggestion
        // PlalistControls should be visible only if there is some item in
```

Reply:
Also should be `PlaylistControls` instead of `PlalistControls`.